README.md

```diff
@@ -1576,4 +1576,3 @@ Two examples of this:
 Version          | Date       | Notes
 ---              | ---        | ---
 1.0.0            | 2019-01-01 | Fork and First release of the OpenRPC Specification
-3.0.2 (upstream) | 2018-10-08 | Patch release of the OpenRPC Specification 3.0.2
```

Comment:
THis isnt extra, its denoting a previous version, 1.0.0 came from 3.0.2 (upstream)

Reply:
If the purpose of that section is to keep a  history of its changes inside the document itself, then IMO its necessary info.

I wouldn't be opposed to just removing it all together though

Reply:
Yeah... Maybe we just remove it? The history is in the git history right?

Reply:
do we REALLY need change notes in this file ?